src/index.ts

```diff
@@ -82,12 +82,12 @@ function loadJavaScript({
 } = {}): webpack.Configuration {
   return {
     resolve: {
-      extensions: [".js"],
+      extensions: [".jsx", ".js"],
```

Comment:
I don't think that we are using the `jsx` extension anywhere - not even in `insights`. I see that our current webpack config in `insights` has it, but I think it is a leftover from earlier times. We moved away from the `.jsx` extension and use `.js` only. We could keep it though, as it doesn't do any harm.

Reply:
Ah, I see. Good to know!

Let's keep it then.